### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by drewvolz**
Minor tweaks to the contributing file.

- Create level I title for document
- Create table of contents for easy navigation
- Convert all subheadings to level II from intermixed level II and III headings
- Correct spelling of "running"

### CODE REVIEW

**Review by rye — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,8 +1,16 @@
+# Contributing
+
 Would you like to contribute? Great! Have a look at [React Native's "getting started" guide](http://facebook.github.io/react-native/docs/getting-started.html) to help you get your computer set up, and make a pull request! (If you feel like you should have write access to the repo, please make an issue and we can discuss it.)
 
 If you are looking to get started with the project, issues tagged with [`good first issue`](https://github.com/StoDevX/AAO-React-Native/issues?q=is%3Aissue+label%3A%22good+first+issue%22+is%3Aopen) are a great place to start. These issues are ones that the team has identified as simple, small changes that won't get too deep, and will be easy to approve for merging.
 
-### Labels
+## Jump to section
```
Comment (hawkrives) [inline CONTRIBUTING.md]: Table of Contents?

Comment (drewvolz) [reply CONTRIBUTING.md]: Would you prefer that?

Comment (hawkrives) [reply CONTRIBUTING.md]: Yes

Comment (drewvolz) [reply CONTRIBUTING.md]: Changed.

**Review by rye — APPROVED** [review]

**Review by hawkrives — APPROVED** [review]

**Pull request merged**